### DISCUSSION TITLE
How do localized content items and localized taxonomy work together?

### DISCUSSION THREAD

**Discussion by juergen-de** [OrchardCMS/OrchardCore#9550] [Q&A]
I have ContentTypes with a localization part like articles and blogposts. As I can only assign a taxonomy ContentItem to a ContentType, I cannot create localized taxonomies or I am forced to create separate ContentTypes for each language which makes localization even more painful to maintain as we have then at least 3 various ways to create and maintain localized content.
To me it seems that to make them work together would require a different approach than assigning a localized ContentItem to a ContentType. Or am I simply doing something wrong?
I tried just using a neutral taxonomy btw which is nice as long as you dont list all items that belong to a taxonomy item. If you do that, you get all ContentItems listed without language selection being honored.

**Comment by Skrypt** [comment]
It definitely needs some development as this part is not working currently. We need to implement localization for Taxonomy terms.
